src/agent/src/device/mod.rs

```diff
@@ -294,15 +294,14 @@ pub async fn handle_cdi_devices(
                 return Ok(());
             }
             Err(e) => {
-                info!(logger, "error injecting devices: {:?}", e);
-                println!("error injecting devices: {:?}", e);
+                info!(logger, "waiting for CDI spec(s) to be generated ({} of {} max tries) {:?}", i, cdi_timeout.as_secs(), e);
```

Comment:
cargo fmt wants this to be split over multiple lines as it's too long

Reply:
Meh, always run `cargo fmt` after chaninging code, always run `cargo fmt` after changing code ....